dotnet-peer/README.md

```diff
@@ -0,0 +1,172 @@
+# .NET libp2p Chat Application
+
+![.NET libp2p](https://img.shields.io/badge/.NET%208.0-libp2p-blue)
+![C# 12](https://img.shields.io/badge/C%23-12.0-brightgreen)
+
+
+<p align="center">
+  <img src="screenshots/dotnet-UI.png" alt=".NET Peer Chat UI"/>
+  <br/>
+  <em>The .NET libp2p chat application Console UI</em>
+</p>
+
+A simple chat application built on top of [dotnet-libp2p](https://github.com/NethermindEth/dotnet-libp2p). This project demonstrates a robust, console-based UI (using [Terminal.Gui](https://github.com/migueldeicaza/gui.cs)) to showcase basic peer-to-peer communication using Libp2p protocols. **Note:** This project is in beta and under active development.
+
+## Overview
+
+dotnet-libp2p Chat aims to provide a performant, well-tested implementation of a wide range of protocols that works on multiple platforms, with high throughput and a low memory profile. This chat application leverages Libp2p's PubSub to exchange messages between peers while also displaying vital information such as your peer ID and multiaddress so that you can easily share your connection details with others.
+
+### Key Features
+- **Console UI:** A robust, multi-tab interface built with Terminal.Gui.
+- **Peer Information:** Displays generated peer ID and multiaddress for connecting with others.
+- **Messaging:** Supports JSON-formatted messages as well as plain text messages.
+- **Protocol Integration:** Uses multiple Libp2p protocols for transport and discovery.
+- **Thread-Safe Updates:** UI updates are marshaled on the main loop ensuring smooth and safe interactions.
+## Requirements
+
+- [.NET 8.0 SDK](https://dotnet.microsoft.com/download/dotnet/8.0) or newer
+- Windows, macOS, or Linux operating system
+
+## Installation
+
+Clone the repository and navigate to the dotnet-peer directory:
+
+```bash
+git clone https://github.com/leny62/universal-connectivity
```

Comment:
Can be changed to the main repo

Reply:
@flcl42 , I'll address these changes ASAP.

@dhuseby , had created this branch `dotnet-peer-development` to merge the .NET to it until its production ready.